src/inference/src/threading/ie_istreams_executor.cpp

```diff
@@ -50,4 +50,8 @@ IStreamsExecutor::Config IStreamsExecutor::Config::MakeDefaultMultiThreaded(cons
     return make_default_multi_threaded(initial);
 }
 
+IStreamsExecutor::Config IStreamsExecutor::Config::ReserveCpuThreads(const Config& initial) {
```

Comment:
Just wondering why reservation became logic of the config? How it is going to be used?

Reply:
It can be provided to other plugin calls for priority use of CPU resources. The main setting parameter is streams_info_talbe in this config.